pkg/master/master.go

```diff
@@ -216,6 +216,10 @@ type Config struct {
 	// Defaults to 6443 if not set.
 	ReadWritePort int
 
+	// The name of the port on PublicAddress and master service port
+	// Defaults to "https" if not set.
+	ReadWritePortName string
```

Comment:
Well, it might not be 443, but I think it will always be "https".

On Wed, Oct 7, 2015 at 9:17 AM, Andrew Butcher <notifications@github.com>
wrote:

> In pkg/master/master.go
> <https://github.com/kubernetes/kubernetes/pull/14871#discussion_r41411085>
> :
>
> > @@ -216,6 +216,10 @@ type Config struct {
> >  	// Defaults to 6443 if not set.
> >  	ReadWritePort int
> >
> > +	// The name of the port on PublicAddress and master service port
> > +	// Defaults to "https" if not set.
> > +	ReadWritePortName string
>
> No, port 443 will always be there so we could just assume it's going to be
> named "https".
> … <#15043179f5722e09_>
> On Wednesday, October 7, 2015, Tim Hockin <notifications@github.com>
> wrote: In pkg/master/master.go <
> https://github.com/kubernetes/kubernetes/pull/14871#discussion_r41410569>
> : > @@ -216,6 +216,10 @@ type Config struct { > // Defaults to 6443 if not
> set. > ReadWritePort int > > + // The name of the port on PublicAddress and
> master service port > + // Defaults to "https" if not set. > +
> ReadWritePortName string Do we really need to parameterize this? — Reply to
> this email directly or view it on GitHub <
> https://github.com/kubernetes/kubernetes/pull/14871/files#r41410569>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/14871/files#r41411085>.
>